products/healthcare/api.yaml

```
@@ -540,3 +540,74 @@ objects:
         'Creating a HL7v2 Store':
           'https://cloud.google.com/healthcare/docs/how-tos/hl7v2'
       api: 'https://cloud.google.com/healthcare/docs/reference/rest/v1/projects.locations.datasets.hl7V2Stores'
+  - !ruby/object:Api::Resource
+    name: 'ConsentStore'
+    kind: "healthcare#consentStore"
+    base_url: "{{dataset}}/consentStores?consentStoreId={{name}}"
+    self_link: "{{dataset}}/consentStores/{{name}}"
+    update_verb: :PATCH
+    update_mask: true
+    min_version: beta
+    description: |
+      The Consent Management API is a tool for tracking user consents and the documentation associated with the consents. 
+    iam_policy: !ruby/object:Api::Resource::IamPolicy
+      method_name_separator: ':'
+      fetch_iam_policy_verb: :GET
+      parent_resource_attribute: 'consent_store_id'
+      import_format: ["{{%dataset}}/consentStores/{{name}}", "{{name}}"]
+      base_url: "{{%dataset}}/consentStores/{{name}}"
+      self_link: "{{%dataset}}/consentStores/{{name}}"
+    parameters:
+      - !ruby/object:Api::Type::ResourceRef
+        name: "dataset"
+        description: |
+          Identifies the dataset addressed by this request. Must be in the format
+          'projects/{project}/locations/{location}/datasets/{dataset}'
+        required: true
+        input: true
+        resource: 'Dataset'
+        imports: 'selfLink'
+        url_param_only: true
+    properties:
+      - !ruby/object:Api::Type::String
+        name: 'name'
+        description: |
+          The last part of the ConsentStore name, for example:
```

```suggestion
          The name of this Consent Store
```

Users (or at least I) generally think of the `name` of a resource as only the last part of the name. The longer form would be either the self_link or `id` field in Terraform